mobile/src/Student/screens/Game/PhaseOneBasicGamePlay/index.js

```diff
@@ -319,17 +319,17 @@ const styles = StyleSheet.create({
         textAlign: 'center',
     },
     hintsViewCard: {
-        marginTop: -40,
-        paddingBottom: scale(20),
+        marginTop: -scale(40),
+        paddingVertical: 0,
         maxHeight: verticalScale(400),
     },
     hintsViewQuestion: {
-        paddingVertical: 0,
+        paddingVertical: scale(30)
```

Comment:
Since it's vertical, you should use `verticalScale` instead. You can read about this library [here](https://www.npmjs.com/package/react-native-size-matters)